### PR TITLE
[13179] Fix doxygen documentation

### PR DESCRIPTION
**Pull request by JLBuenoLopez-eProsima**
Signed-off-by: JLBuenoLopez-eProsima <joseluisbueno@eprosima.com>

### CODE REVIEW

File: include/fastdds/rtps/history/ReaderHistory.h [hunk 1]
```diff
@@ -77,6 +77,8 @@ class ReaderHistory : public History
      * In this implementation this method just calls add_change. The user can overload this method in case
      * he needs to perform additional checks before adding the change.
      * @param change Pointer to the change
+     * @param unknown_missing_changes_up_to The number of changes from the same writer with a lower sequence number that
+     *                                     could potentially be received in the future.
```
Comment (MiguelCompany) [inline include/fastdds/rtps/history/ReaderHistory.h:81]:
```suggestion
     * @param unknown_missing_changes_up_to The number of changes from the same writer with a lower sequence number that
     *                                      could potentially be received in the future.
```

**Review by MiguelCompany — CHANGES_REQUESTED** [review]
Just a minor request

**Review by MiguelCompany — APPROVED** [review]

**Review by MiguelCompany — APPROVED** [review]
LGTM

File: include/fastdds/rtps/common/InstanceHandle.h [hunk 2]
```diff
@@ -287,9 +303,11 @@ inline std::ostream& operator <<(
 }
 
 /**
+ * Stream operator: retrieve an InstanceHandle_t.
  *
- * @param input
- * @param iHandle
+ * @param input Input stream.
+ * @param iHandle InstanceHandle_t to print.
```
Comment (jsantiago-eProsima) [inline include/fastdds/rtps/common/InstanceHandle.h:309]:
```suggestion
 * @param iHandle InstanceHandle_t that will receive the input as its new value.
```

**Review by jsantiago-eProsima — CHANGES_REQUESTED** [review]
Just a comment

**Review by jsantiago-eProsima — APPROVED** [review]
LGTM

**Pull request merged**